src/locale/az.js

```diff
@@ -0,0 +1,39 @@
+import dayjs from 'dayjs'
+
+const locale = {
+  name: 'az',
+  weekdays: 'Bazar_Bazar ertəsi_Çərşənbə axşamı_Çərşənbə_Cümə axşamı_Cümə_Şənbə'.split('_'),
+  weekdaysShort: 'Baz_BzE_ÇAx_Çər_CAx_Cüm_Şən'.split('_'),
+  weekdaysMin: 'Bz_BE_ÇA_Çə_CA_Cü_Şə'.split('_'),
+  months: 'yanvar_fevral_mart_aprel_may_iyun_iyul_avqust_sentyabr_oktyabr_noyabr_dekabr'.split('_'),
+  monthsShort: 'yan_fev_mar_apr_may_iyn_iyl_avq_sen_okt_noy_dek'.split('_'),
```

Comment:
Thanks for adding the az locale. I think these should have first letter capitalized. For example mantine date picker uses dayjs for date formatting and az locale looks like this:

<img width="413" alt="Screen Shot 2023-03-27 at 1 28 16 AM" src="https://user-images.githubusercontent.com/15037839/227805656-ebf604fc-1c44-4ea2-9d97-7c4dbe6a6647.png">

Note how text input is `mart` in lowercase. I'm not sure how I can back my claim up but [Turkish](https://github.com/iamkun/dayjs/blob/dev/src/locale/tr.js#L9-L10) locale has the them capitalized.